llvm/test/Transforms/SLPVectorizer/SystemZ/vec-elt-insertion.ll

```diff
@@ -0,0 +1,88 @@
+; RUN: opt -S --passes=slp-vectorizer -mtriple=s390x-unknown-linux -mcpu=z16 < %s | FileCheck %s
```

Comment:
Better precommit the test with the current codegen. Also, would be good to add cost estimations check to show that is modified

Reply:
1. The test itself better to precommit in a separate NFC patch anyway.
2. There should be the difference, otherwise it is meaningless. No need to check other passes, SLP vectorizer has its own cost estimation, which can be obtained via `-pass-remarks-output=` options. There are several tests already, which use it.

Reply:
ok - there is indeed a difference with the pass-remarks-output which is quite readable. Test updated.